packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Accordion/accordion';
+import {TitleLevel} from '../Title';
```

Comment:
You don't need to import this here, it's not being used in the component itself.